bili.config.ts

```diff
@@ -8,10 +8,16 @@ const config: Config = {
     format: ['cjs', 'umd', 'umd-min', 'esm'],
     moduleName: 'ReactEasyCrop',
     sourceMap: true,
+    extractCSS: false,
```

Comment:
could you explain a bit what these 2 new options are doing please ? 🙂

Reply:
By default the built tool bili extract css imports to its own files and output then in the dist folder, this disabled that feature,

After disabling it, the bili rollup configuration is set to automaticly inject the style tag in the head of the dom, this postcss config disables that, we needed to disabled that feature else the rollup script would end up inside the bundle